### PR TITLE
Add Client Link-Layer Address option for RFC6939 support

### PR DESCRIPTION
**Pull request by mcphailtom**

### CODE REVIEW

**Comment by pmazzini** [comment]
Thanks for the PR!

File: dhcpv6/dhcpv6relay.go [hunk 1]
```diff
@@ -3,6 +3,7 @@ package dhcpv6
 import (
 	"errors"
 	"fmt"
+	"github.com/insomniacslk/dhcp/iana"
```
Comment (pmazzini) [inline dhcpv6/dhcpv6relay.go:6]: Better to keep it consistent with other files and have this import in the 2nd block
https://github.com/insomniacslk/dhcp/blob/master/dhcpv6/dhcpv6.go#L8

File: dhcpv6/option_clientlinklayeraddress_test.go [hunk 2]
```diff
@@ -0,0 +1,40 @@
+package dhcpv6
+
+import (
+	"bytes"
+	"github.com/insomniacslk/dhcp/iana"
```
Comment (pmazzini) [inline dhcpv6/option_clientlinklayeraddress_test.go:5]: Better to keep it consistent with other files and have this import in the 2nd block
https://github.com/insomniacslk/dhcp/blob/master/dhcpv6/dhcpv6.go#L8

File: dhcpv6/option_clientlinklayeraddress.go [hunk 3]
```diff
@@ -0,0 +1,46 @@
+package dhcpv6
+
+import (
+	"fmt"
+	"github.com/insomniacslk/dhcp/iana"
```
Comment (pmazzini) [inline dhcpv6/option_clientlinklayeraddress.go:5]: Better to keep it consistent with other files and have this import in the 2nd block
https://github.com/insomniacslk/dhcp/blob/master/dhcpv6/dhcpv6.go#L8

**Comment by mcphailtom** [comment]
updated 👍

**Comment by pmazzini** [comment]
Linters are failing, I have a PR to fix it #383.

File: dhcpv6/dhcpv6relay.go [hunk 4]
```diff
@@ -54,6 +55,18 @@ func (ro RelayOptions) RemoteID() *OptRemoteID {
 	return nil
 }
 
+// ClientLinkLayerAddress returns the Hardware Type and Link Layer Address of the requesting client in this relay message.
```
Comment (pmazzini) [inline dhcpv6/dhcpv6relay.go:58]: NIT: this comment is too long, can you add a line break

File: dhcpv6/modifiers.go [hunk 5]
```diff
@@ -152,3 +152,9 @@ func WithIAPD(iaid [4]byte, prefixes ...*OptIAPrefix) Modifier {
 		}
 	}
 }
+
+// WithClientLinkLayerAddress adds or updates the ClientLinkLayerAddress option with provided HWType and HWAddress on a
```
Comment (pmazzini) [inline dhcpv6/modifiers.go:156]: NIT: this comment is too long, can you add a line break

**Comment by mcphailtom** [comment]
> lgtm

Thanks 👍

**Comment by insomniacslk** [comment]
Thanks! Can you please rebase on #383?

**Comment by mcphailtom** [comment]
Thanks @pmazzini ... had to have some sleep.. appreciate the rebase 👍

**Pull request merged**